guides/common/assembly_preparing-environment-for-installation.adoc

```diff
@@ -23,5 +23,7 @@ include::modules/proc_enabling-client-connections-to-satellite.adoc[leveloffset=
 
 include::modules/proc_verifying-dns-resolution.adoc[leveloffset=+1]
 
+include::modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+1]
```

Comment:
This was originally under `ifdef::katello,orcharhino,satellite`. Perhaps the ifdef should be also here? @ekohl